shared/macros.jinja

```diff
@@ -618,10 +618,42 @@ ocil_clause: "the correct value is not returned"
 
 
 {{% macro body_of_warning_about_dependent_rule(rule_id, why) -%}}
-        When selecting this rule in a profile, 
+        When selecting this rule in a profile,
         {{%- if why %}}
             make sure that rule with ID <code>{{{ rule_id }}}</code> is selected as well: {{{ why }}}
         {{%- else %}}
             rule <code>{{{ rule_id }}}</code> has to be selected as well.
         {{%- endif %}}
 {{% endmacro %}}
+
+{{% macro openssl_strong_entropy_config_file() -%}}
```

Comment:
What is the reason to define a macro? Wouldn't a variable suffice?

Reply:
The variable could be theoretically used, but you would need to take care about escaping characters, in particular ' or " because you use one of those characters to enclose the variable value. Macro is foolproof in this case.

Reply:
Ok, that is an interesting side effect of using macros.
Long term wise, I'm not sure if we should rely on this aspect.